src/map/data.ts

```diff
@@ -1,56 +1,23 @@
 import Fuse from 'fuse.js'
 import querystring, { ParsedUrlQueryInput } from 'querystring'
 import { RawMapCollection } from '../generated/map-collection'
-import { LegendItem, RawMapLayer } from '../generated/map-layer'
-import { RawTheme } from '../generated/theme'
+import { LegendItem as RawLegendItem, RawMapLayer } from '../generated/map-layer'
+import { Theme } from '../generated/theme'
+import {
+  MapLayer as ComposedMapLayer,
```

Comment:
I think we can start using the name `MapLayer` here directly instead of `ComposedMapLayer`, the same goes for `MapCollection`.